app/src/main/res/values-sw360dp/auto_dimens.xml

```diff
@@ -0,0 +1,22 @@
+<resources>
+    <dimen name="sp_23">23sp</dimen>
+    <dimen name="dp_200">10dp</dimen>
+    <dimen name="dp_10">10dp</dimen>
+    <dimen name="sp_13">13sp</dimen>
+    <dimen name="dp_170">170dp</dimen>
+    <dimen name="sp_17">17sp</dimen>
+    <dimen name="dp_15">15dp</dimen>
+    <dimen name="dp_12">12dp</dimen>
+    <dimen name="sp_20">20sp</dimen>
+    <dimen name="dp_16">16dp</dimen>
+    <dimen name="sp_6">6sp</dimen>
+    <dimen name="sp_18">18sp</dimen>
+    <dimen name="dp_36">36dp</dimen>
+    <dimen name="dp_26">26dp</dimen>
+    <dimen name="dp_54">54dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_54` appears to be unused
 Unused resources make applications larger and slow down builds.